src/OpenApi/Model/Link.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\OpenApi\Model;
+
+class Link
+{
+    use ExtensionTrait;
+
+    // public $operationRef;
```

Comment:
?

Reply:
> A relative or absolute reference to an OAS operation. This field is mutually exclusive of the operationId field, and MUST point to an Operation Object. Relative operationRef values MAY be used to locate an existing Operation Object in the OpenAPI definition.

This is a required field according to the spec but I'm not sure what to put in it :D